packages/cice5/package.py

```diff
@@ -97,14 +118,22 @@ def edit(self, spec, prefix):
         config = {}
 
         istr = join_path((spec["oasis3-mct"].headers).cpp_flags, "psmile.MPI1")
-        ideps = ["parallelio", "oasis3-mct", "libaccessom2", "netcdf-fortran"]
+        ideps = ["oasis3-mct",  "netcdf-fortran"]
         incs = " ".join([istr] + [(spec[d].headers).cpp_flags for d in ideps])
 
-        lstr = self.make_linker_args(spec, "parallelio", "-lpiof -lpioc")
-        # NOTE: The order of the libraries matter during the linking step!
-        # NOTE: datetime-fortran is a dependency of libaccessom2.
-        ldeps = ["oasis3-mct", "libaccessom2", "netcdf-c", "netcdf-fortran", "datetime-fortran"]
+        ldeps = ["oasis3-mct", "netcdf-c", "netcdf-fortran"] 
+        lstr = str()
+
+        if self.spec.satisfies("master"):
+            lstr = self.make_linker_args(spec, "parallelio", "-lpiof -lpioc")
+            ideps.extend(["libaccessom2"])
+
+            # NOTE: The order of the libraries matter during the linking step!
+            # NOTE: datetime-fortran is a dependency of libaccessom2.
+            ldeps = ["oasis3-mct", "libaccessom2", "netcdf-c", "netcdf-fortran", "datetime-fortran"]
+
         libs = " ".join([lstr] + [self.get_linker_args(spec, d) for d in ldeps])
+        libs = " ".join([self.get_linker_args(spec, d) for d in ldeps])
```

Comment:
Follow the pattern in the MOM5 SPR: https://github.com/ACCESS-NRI/spack-packages/blob/ff2fbc0a9680118a805a0577edbe03008c5346ae/packages/mom5/package.py#L67-L87

Reply:
Resolved in https://github.com/ACCESS-NRI/spack-packages/pull/179